app/src/main/java/org/mozilla/focus/locale/screen/LanguageFragment.kt

```diff
@@ -5,71 +5,68 @@
 package org.mozilla.focus.locale.screen
 
 import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
 import androidx.compose.foundation.lazy.rememberLazyListState
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.MutableState
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.rememberCoroutineScope
-import androidx.compose.ui.platform.ComposeView
 import kotlinx.coroutines.launch
+import mozilla.components.browser.state.store.BrowserStore
 import mozilla.components.lib.state.ext.observeAsComposableState
 import mozilla.components.support.locale.LocaleUseCases
 import org.mozilla.focus.R
 import org.mozilla.focus.ext.components
 import org.mozilla.focus.ext.requireComponents
-import org.mozilla.focus.settings.BaseSettingsComposeFragment
+import org.mozilla.focus.settings.BaseComposeFragment
 import org.mozilla.focus.state.AppAction
 import org.mozilla.focus.state.Screen
 
-class LanguageFragment : BaseSettingsComposeFragment() {
-    private val browserStore by lazy { requireContext().components.store }
-    private val localeUseCase by lazy { LocaleUseCases(browserStore) }
-    private val languageScreenStore by lazy {
-        LanguageScreenStore(
+class LanguageFragment : BaseComposeFragment() {
+
```

Comment:
nit: I do not think the space is necessary and we d not have it in other fragments.